rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/20_unsupported_apis.yml

```diff
@@ -65,11 +65,6 @@
       indices.create:
         index: logs-foobarbaz
 
-  - do:
```

Comment:
I'll mute that portion of the test for now since that one call has different behavior when security is enabled or not. Once this PR and https://github.com/elastic/elasticsearch/pull/57900 land, its behavior will be the same.